### ISSUE TITLE
error: Handshake status 403 forbidden

### ISSUE THREAD

**Issue by fragmental** [Paco8/SkyExtractCookie#2]
This is the error I get when running the error when I run the program:
```
Would you like to extract the cookie for PeacockTV or SkyShowTime?
1) PeacockTV
2) SkyShowtime
1
Browser startup... (C:\Program Files (x86)\Google\Chrome\Application\chrome.exe) please wait
Establish connection with the browser in port 9222... please wait
An error occurred:
Handshake status 403 Forbidden
Traceback (most recent call last):
  File "SkyExtractCookie.py", line 85, in __init__
  File "SkyExtractCookie.py", line 105, in operations
  File "SkyExtractCookie.py", line 152, in get_browser_debug_endpoint
  File "websocket\_core.py", line 594, in create_connection
  File "websocket\_core.py", line 253, in connect
  File "websocket\_handshake.py", line 79, in handshake
  File "websocket\_handshake.py", line 165, in _get_resp_headers
websocket._exceptions.WebSocketBadStatusException: Handshake status 403 Forbidden

Press "Enter" to finish
```

Edit:  possibly related to #1 as the same thing happens. I'm on windows 10 x64

**Comment by Paco8** [comment]
Maybe the firewall is preventing the conection?
Anyway, I'm developing an android app which I think it would be better than an application for PC, could you test if it works for you?
https://github.com/Paco8/SkyExtractCookieAndroid

**Comment by fragmental** [comment]
Idk about the firewall blocking it.  It should prompt me, before trying to block anything, but I can investigate it.

 I tried the Android app and it worked well.  Since I already have a setup to send files from my Android phone to my Chromecast with Google TV HD using the Send to TV apps, the process was relatively quick and painless and the key file worked with no issue.

After I confirmed the key filed worked I also went ahead and sent the apk over to my Chromecast, installed it, ran the app, after finding it in the settings, and it seemed to work also.  However, I did have to change the file permission for files and media to "allow all the time", or it would fail when trying to save the file.  I was familiar with that process because I needed to do the same thing with Kodi in order to install the addon from a zip file.
